src/js/modules/PageBreadcrumbs.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import I18nNavLink from 'js/modules/I18nNavLink';
+
+const PageBreadcrumbs = (props) => {
+
+  const { title, order, ...rest } = props;
```

Comment:
could this variable destructuring happen inline on line 4?

Reply:
YEP. updating in my next branch.